weather/weather.go

```diff
@@ -131,18 +154,30 @@ func (p *Parser) Parse(values url.Values) {
 
 	updateGauge(p.battery.WithLabelValues(p.name, "outdoor"))(parseValue("battout"))
 	updateGauge(p.battery.WithLabelValues(p.name, "indoor"))(parseValue("battin"))
+	updateGauge(p.battery.WithLabelValues(p.name, "lightning"))(parseValue("batt_lightning"))
 	updateGauge(p.humidity.WithLabelValues(p.name, "indoor"))(parseValue("humidityin"))
 	updateGauge(p.barometer.WithLabelValues(p.name, "relative"))(parseValue("baromrelin"))
 	updateGauge(p.barometer.WithLabelValues(p.name, "absolute"))(parseValue("baromabsin"))
-	updateGauge(p.windDir.WithLabelValues(p.name))(parseValue("winddir"))
+	updateGauge(p.windDir.WithLabelValues(p.name, "current"))(parseValue("winddir"))
+	updateGauge(p.windDir.WithLabelValues(p.name, "avg10m"))(parseValue("winddir_avg10m"))
```

Comment:
I would think that average direction should be left as an exercise to the metrics collector, e.g. prometheus or victoriametrics queries with `avg_over_time(wind_dir[10m])`